src/tools/genpolicy/src/registry.rs

```diff
@@ -508,3 +628,31 @@ fn build_auth(reference: &Reference) -> RegistryAuth {
 
     RegistryAuth::Anonymous
 }
+
+fn parse_passwd_file(passwd: String) -> Result<Vec<PasswdRecord>> {
```

Comment:
I just enabled tests for genpolicy the other day - how about adding one for this parser?